jwst/jump/jump.py

```diff
@@ -73,11 +73,13 @@ def detect_jumps(input_model, gain_model, readnoise_model,
     if len(wh_g[0] > 0):
         pdq[wh_g] = np.bitwise_or(pdq[wh_g], dqflags.pixel['NO_GAIN_VALUE'])
         pdq[wh_g] = np.bitwise_or(pdq[wh_g], dqflags.pixel['DO_NOT_USE'])
+        gain_2d[wh_g] == 0
```

Comment:
Should be just a single "=" sign here, right? You want to set the values to zero, not evaluate them.